phpcs.xml.dist

```diff
@@ -1,62 +1,81 @@
 <?xml version="1.0"?>
 <ruleset name="WP Job Manager">
-	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
-	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
-
-	<!-- Set a description for this ruleset. -->
 	<description>A custom set of code standard rules to check for WordPress themes and plugins.</description>
 
-	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs,vendor/phpcompatibility/php-compatibility" />
+	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs,vendor/phpcompatibility/phpcompatibility-paragonie,vendor/phpcompatibility/php-compatibility,vendor/phpcompatibility/phpcompatibility-wp,vendor/sirbrillig/phpcs-variable-analysis" />
```

Comment:
If you add https://packagist.org/packages/dealerdirect/phpcodesniffer-composer-installer to your require-dev, then standards will get automatically registered on `composer update` and `composer install`. No need for this `installed_paths` config.

Reply:
We already use that package. I'm not sure why we maintain it here as well. I can remove it.